source/community/policy/editorial.md

```diff
@@ -34,9 +36,9 @@ In order to be considered ready for final review, new features must have two ope
 
 New versions of specifications at or above 1.0 must be reviewed by the [Technical Review Committee (TRC)][trc] as representative of the consortium and the community with regards to technical matters. Both the agendas and the notes from TRC meetings will be openly available.
```

Comment:
and Beta versions of spec? Related to content state discussions and also we took change notification to 0.9

Reply:
What about minor releases after Beta? Thinking about the small changes that were brought up by the TRC and results in a 0.9.1. We discussed sub versions don't need to go to TRC only larger changes e.g. 0.9.1 wouldn't go but 1.0 would.

Reply:
I think @azaroth42 's addition of "at least beta" in 3752f11 fixes this